tests/roles/dataplane_adoption/tasks/nova_ffu.yaml

```diff
@@ -95,18 +81,38 @@
       nodeSets:
         - openstack
       servicesOverride:
-        - nova-compute-ffu
+        - nova-compute-extraconfig
     EOF
 
 - name: wait for Nova compute EDPM services to become ready
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    oc wait --for condition=Ready osdpd/openstack-nova-compute-ffu --timeout=5m
-  register: nova_ffu_edpm_result
-  until: nova_ffu_edpm_result is success
-  retries: 10
-  delay: 6
+
+    DEPLOYMENT_NAME=openstack-nova-compute-ffu
```

Comment:
I am not sure why just waiting for the ready status condition is no longer  sufficient.
If we really have to, please adjust the docs to follow this change

Reply:
the reason was at least for @vakwetu the wait can happen and fail before the deployment is created


so if we use the wait we need to extend the time interval

i.e. it took time for the nodeset to be reconsolied for the changed config map before the deployment was created so all the reties were used before the resource was available to wait on